ravenframework/Models/HybridModels/HybridModel.py

```diff
@@ -178,6 +179,11 @@ def initialize(self,runInfo,inputs,initDict=None):
       if romIn.amITrained:
         self.raiseAWarning("The provided rom ", romIn.name, " is already trained, we will reset it!")
         romIn.reset()
+        # reset HybridModel attributes to allow workflow re-running
+        self.romConverged = False
+        self.oldTrainingSize = 0
+        self.modelIndicator = {}
+        self.crowdingDistance = None
```

Comment:
could we collect this in a particular reset method so it's obvious where to expand these if we need to in the future?

Reply:
I created a new `resetHybridModel` method that encapsulates this.